Sources/Components/StepSlider/StepSlider.swift

```diff
@@ -80,6 +90,19 @@ public final class StepSlider: UISlider {
         updateActiveTrackRange()
     }
 
+    public override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        super.traitCollectionDidChange(previousTraitCollection)
+
+        guard
+            #available(iOS 12.0, *),
```

Comment:
Why do we need this availability check here?

Reply:
`userInterfaceStyle` is iOS 12+